rma_reason/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Raumschmiede GmbH
+# Copyright 2024 BCIM
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Rma Reason",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Raumschmiede GmbH,BCIM,ACSONE SA/NV,Odoo Community Association (OCA)",
```

Comment:
I think `Raumschmiede GmbH` should go in the CREDITS file, not as author, being the paying customer, isn't it?

Reply:
@jbaudoux , fine with you?